css/style.css

```diff
@@ -568,4 +572,472 @@ h3.lang-1 {
+    display: none;
+  }
+
+  .mobile-snapshoot-4 {
+    display: none;
+  }
+
+  a.facebook.facebook-2 {
+    display: none;
+  }
+
+  a.back.back-desktop.back-desktop-2 {
+    display: none;
+  }
+
+  p.work-text-2 {
+    font-family: Poppins-Thin, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    line-height: 24px;
+    color: #344563;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 12px;
+  }
+
+  p.work-text-desktop {
+    display: none;
+  }
+
+  .btn-ruby {
+    display: none;
+  }
+
+  .buttons-group ul li {
+    padding-right: 8px;
+  }
+
+  .desktop-snapshoot-1-3 {
+    display: none;
+  }
+
+  .desktop-snapshoot-2 {
+    display: none;
+  }
+
+  h2.desktop-text {
+    display: none;
+  }
+
+  a.facebook-desktop {
+    display: none;
+  }
+
+  a.back.back-desktop {
+    display: none;
+  }
+
+  p.work-text-3 {
+    font-family: Poppins-Thin, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    line-height: 24px;
+    color: #344563;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 12px;
+  }
+
+  .snp-4 {
+    display: block;
+  }
+
+  a.canopy.uber {
+    display: none;
+  }
+
+  .back-lead {
+    display: none;
+  }
+
+  .year-more {
+    display: none;
+  }
+
+  p.work-text-4 {
+    font-family: Poppins-Thin, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    line-height: 24px;
+    color: #344563;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 12px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  header {
+    padding: 16px 143px;
+    background-color: #fff;
+  }
+
+  .hamb-icon {
+    display: none;
+  }
+
+  ul.nav-items {
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .list-card-primary li a {
+    padding-right: 10px;
+  }
+
+  ul.nav-items li a {
+    text-decoration: none;
+    padding: 10px 12px;
+    align-items: center;
+    gap: 10px;
+    font-family: Poppins-Regular, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .desktop-version {
+    padding-top: 289px;
+    padding-right: 20%;
+    padding-left: 20%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    gap: 12px;
+    background: #fff no-repeat right/87% 98% url(../img/header-bg@2x.png);
+    border-bottom-left-radius: 80px 80px;
+  }
+
+  .presentation-part h1 {
+    width: 100%;
+    letter-spacing: 0.37px;
+    flex-direction: column;
+    justify-content: center;
+    font-family: Poppins-Regular, sans-serif;
+    font-size: 48px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 60px;
+    color: #091e42;
+    padding-right: 0;
+    padding-left: 0;
+  }
+
+  p.presentation-paragraph {
+    font-family: Poppins-Thin, sans-serif;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    padding-right: 0;
+    padding-left: 0;
+  }
```

Comment:
- [ ] Great work for making the desktop version of your portfolio. You've really put in great works styling your desktop version without affecting your mobile. Kudos to you man :handshake: . There's just little stufff you need to work on in your about me section. According to the Figma design, the paragraph part takes the width of 40% and the language list takes the width of 60% as opposed to your design. Kindly look into the two screenshots and walk your design alongside the Figma design.

### Your Design
![your-design](https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/26597432/0bd74d40-cccc-44c9-86ea-52c009afb70d)

### Figma Design
![temp-design](https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/26597432/f63b8101-4951-492e-8e55-e8127c9d965a)

Reply:
Dear @Qoosim  👋🏾
Very nice to meet you, thank you for your kind words, and suggestion, I appreciate it.
I modified the width to 40% for the about-myself-first class which contains : the title, the paragraph, link , social icons and buttons. Besides, I gave also a width of 60% to language-list class as you suggested.
I have to mention that all these elements are inside a section named about-myself.
Thanks again,  can you please verify it?
Best regards,